packages/test-support/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@agoric/test-support",
+  "version": "0.1.0",
+  "description": "Support libraries for testing on chain",
```

Comment:
the name `test-support` sounds much broader than that.

these also don't test on a chain, right? Just a swingset.

I request that you leave these in `boot` and export them under its `tools`. `@agoric/benchmark` can import them from there.

If that complicates some other plans, let's chat

Reply:
Yes, this would complicate other plans.  There's little in those files that specifically pertains to the bootstrap tests, but they do contain the current best varsions of some bits that are widely replicated throughout our suite of tests in a copy-paste-edit reuse with minor changes pattern.  It's our hope to start cleaning that stuff up using common implementations of commonly used test boilerplate, which will want to draw on a common package for that stuff.

Reply:
@turadg and I had a long chat about this.  He feels quite strongly that the stuff about setting up a mock chain really belongs in the `boot` package since that's really what that package is all about.  I'm not 100% sure I agree, but on the other hand I don't really have a very good counterargument either.  We did agree that if the stuff is being used from the outside the package it does not belong buried layers down in the `test` directory.  Instead, we're putting the support infra into the `tools` directory.  The pieces of support code that I had flagged as being much more widely useful beyond just the bootstrap tests pertain to convenience sugaring for tests invoking the controller.  This stuff has been broken out into a separate `run-utils` module and placed into `SwingSet/tools`, where it will be available to tests that are using SwingSet but not entraining the rest of the chain machinery.  The `test-support` package is now gone.  I will revise the PR title and descriptive commentary to match this resolution.

I think we still have some confusion to sort out with respect to the somewhat conflicted meaning of the `tools` directory, as it we have used it to hold both "ancillary executables that are useful in a standalone mode" as well as "utility code that is outside the supported package APIs", sometimes even within the scope of a single package.  However, this PR is not the place to resolve that.